mappings/net/minecraft/world/gen/noise/NoiseRouterData.mapping

```diff
@@ -0,0 +1,63 @@
+CLASS net/minecraft/unmapped/C_ronqvany net/minecraft/world/gen/noise/NoiseRouterData
+	FIELD f_craftepc VEIN_FREQUENCY D
+	FIELD f_hfbhqfdm ORE_THICKNESS F
+	FIELD f_ljabpota BLENDING_JAGGEDNESS Lnet/minecraft/unmapped/C_kzwapyud;
+	FIELD f_skugmypg BLENDING_FACTOR Lnet/minecraft/unmapped/C_kzwapyud;
+	FIELD f_sywrklcc NOODLE_SPACING_AND_STRAIGHTNESS D
+	FIELD f_tqarabvz SURFACE_DENSITY_THRESHOLD D
+	METHOD m_bxikrtou noodleCave ()Lnet/minecraft/unmapped/C_kzwapyud;
+	METHOD m_dqlsbhbi finalDensity (Lnet/minecraft/unmapped/C_ulvgadxi;ZZZ)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 config
+	METHOD m_duxzadwj spaghettiCaveRoughnessFunction ()Lnet/minecraft/unmapped/C_kzwapyud;
```

Comment:
Why is this the only ` DensityFunction` factory suffixed with ` Function`?

Reply:
Good question, I'll fix that